telescope/getters/kubernetes.py

```diff
@@ -30,32 +32,47 @@ def get(self, cmd: Union[List[str], str]) -> Union[dict, str]:
         """Utilize kubernetes python client to exec in a container
         https://github.com/kubernetes-client/python/blob/master/examples/pod_exec.py
         """
-        try:
-            pod_res = kube_client.read_namespaced_pod(name=self.name, namespace=self.namespace)
-            if not pod_res or pod_res.status.phase == "Pending":
-                raise RuntimeError(
-                    f"Kubernetes pod {self.name} in namespace {self.namespace} does not exist or is pending..."
-                )
-        except ApiException as e:
-            if e.status != 404:
-                raise RuntimeError(f"Unknown Kubernetes error: {e}")
+        if os.getenv("TELESCOPE_KUBERNETES_METHOD", "") == "kubectl":
+            if type(cmd) == list:
+                cmd = shlex.join(cmd)
 
-        log.debug(f"Running {cmd} on pod {self.name} in namespace {self.namespace} in container {self.container}")
-        exec_res = stream(
-            kube_client.connect_get_namespaced_pod_exec,
-            name=self.name,
-            namespace=self.namespace,
-            command=cmd,
-            container=self.container,
-            stderr=True,
-            stdin=False,
-            stdout=True,
-            tty=False,
-        )
-        log.debug(f"Got output: {exec_res}")
+            cmd = f"kubectl exec -it -n {self.namespace} {self.name} -c {self.container} -- {cmd}"
+            print(cmd)
+            out = run(cmd, hide=True, warn=True).stdout  # other options: timeout, warn
+            out = clean_airflow_report_output(out)
```

Comment:
why do you override out like this, might be more readable to use different variable names here